core/src/main/scala/org/dbpedia/extraction/mappings/ContributorExtractor.scala

```diff
@@ -30,7 +30,7 @@ class ContributorExtractor( context : {
 
     if(node.contributorID <= 0) return Seq.empty
 
-    val pageURL = "http://" + context.language.wikiCode + ".wikipedia.org/wiki/" + node.title.encoded;
+    val pageURL = context.language.baseUri + "/wiki/" + node.title.encoded;
```

Comment:
Can you save the language in a class variable? We had some issues when we were accessing `context ` members inside the `extract()` function from Spark